src/main/java/pl/xkoem/GameConfiguration.java

```diff
@@ -9,6 +9,14 @@
     private Integer sizeY;
     private String language;
 
+    GameConfiguration() {
+        this(3,3,3);
+    }
+
+    GameConfiguration(Integer sizeX, Integer sizeY, Integer charsToWin) {
+        setBoardSettings(sizeX, sizeY, charsToWin);
+    }
+
     public void setBoardSettings(Integer sizeX, Integer sizeY, Integer charsToWin) throws InvalidParameterException {
         if(charsToWin > sizeX || charsToWin > sizeY) {
```

Comment:
Jesteś pewny tego warunku?
Jak masz trójkąt prostokątny o przyprostokątnych 3 i 4, to przeciwprostokątna ma 5.
Więc na planszy 3x4 da się teoretycznie wygrać jak masz 5 znaków na zwycięztwo

Reply:
Nie da sie, gdyż nie patrzymy na przeciwprostokatna tylko na pola po skosie